querybook/webapp/components/DataDocSchedule/DataDocScheduleForm.tsx

```diff
@@ -284,6 +296,161 @@ export const DataDocScheduleForm: React.FunctionComponent<
     );
 };
 
+const NotifactionFormRow: React.FC<{
+    name: string;
+    index: number;
+    removeRow: (index: number) => void;
+    notificationRow: IDataDocScheduleNotification;
+    notifierOptions: string[];
+    notifyOnOptions: IOptions;
+    getHelp: (notifierName: string) => string;
+}> = ({
+    name,
+    index,
+    notificationRow,
+    removeRow,
+    notifierOptions,
+    notifyOnOptions,
+    getHelp,
+}) => {
+    const notificationFormName = `${name}[${index}]`;
+    const handleRemoveNotifier = useCallback(
+        () => removeRow(index),
+        [removeRow, index]
+    );
+    const [_, notifyToAllMeta, notifyToAllHelpers] = useField(
```

Comment:
you can get the first parameter as value, instead of using notificationRow. Its just 1 less prop to worry about
```js
const [{ value: notificationRow }, _, __] = useField